frb_codegen/src/config/opts_parser.rs

```diff
@@ -87,14 +92,67 @@ pub fn config_parse(mut raw: RawOpts) -> Vec<Opts> {
         );
     }
 
+    // rust/dart shared output path, used
+    let (shared_rust_output_path, shared_dart_output_path) = if raw.rust_input.len() == 1 {
```

Comment:
nit: I hope this named variable can make the intention a bit cleaner

```suggestion
    let single_block_mode = raw.rust_input.len() == 1;
    let (shared_rust_output_path, shared_dart_output_path) = if single_block_mode {
```